apps/src/applab/designMode.js

```diff
@@ -359,12 +359,18 @@ designMode.updateProperty = function(element, name, value) {
     case 'picture':
       originalValue = element.getAttribute('data-canonical-image-url');
       element.setAttribute('data-canonical-image-url', value);
+      var cacheBustSuffix = '';
+      if (timestamp) {
```

Comment:
Does it make sense to always use the timestamp in the file name?

Reply:
Sorry, my question was pretty ambiguous. Shouldn't images always use the cacheBustSuffix? i.e. if there is no timestamp, should we create one?

Reply:
Ah sorry, I misunderstood.
I think the timestamp should always be present in the asset, and then it's just plumbed through so it should always be present here as well. I'm inclined to keep this as similar as possible to how it's done in AssetThumbnail. What do you think?

Reply:
Took a closer look at Chris's PR. I think Chris put 'if(timestamp)' into his change because images uploaded before then wouldn't have timestamps. I agree with you, we should do this the same as how it's done in AssetThumbnail. There probably are still a lot of images from before April 2018.